.flake8

```diff
@@ -0,0 +1,6 @@
+# Black-compatible flake8 config
```

Comment:
@KietTruongSFDO I think this file was accidentally committed. If so, would you mind removing from the repo?

Reply:
I didn't add this file, it might just be the branches are out of sync. I'll pull and re-update this PR.